extensions/shared/blocks/cover/components.js

```diff
@@ -0,0 +1,64 @@
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import UpgradeNudge from "../../components/upgrade-nudge";
```

Comment:
Would you mind checking for single quotes again across the new files?

Reply:
✅ I've had to re-implement it using a hook, which I think fits better to the code. 👍